src/common/graph.py

```diff
@@ -115,3 +115,15 @@ def compiled_graph(self) -> CompiledStateGraph:
                 name=self._name, checkpointer=self._checkpointer, store=self._store
             )
         return self._compiled_graph
+
+    async def ainvoke(self, state: Any, config: RunnableConfig | None = None):
```

Comment:
The method isn't needed. The only reason I added it was to auto-inject config values but langgraph studio will _not_ use this method and instead relies on the underlying `compiled_graph`, so to be consistent with the tooling we also need to use the `compiled_graph` and pass in the config manually.

See: https://github.com/Moonsong-Labs/ai-nexus/blob/main/src/demo/orchestrate.py#L121